.travis.yml

```diff
@@ -28,6 +28,7 @@ matrix:
   allow_failures:
     - rvm: rbx-2
     - rvm: jruby
+    - rvm: ruby-head
```

Comment:
`ruby-head` is also required in the`rvm` section. `ruby-head` is not running as follows.

https://travis-ci.org/octokit/octokit.rb/builds/181598215